client/src/app/containers/LeftDrawer/LoggedInDrawer.jsx

```diff
@@ -6,6 +6,7 @@ import ListItem from '@material-ui/core/ListItem';
 import ListItemIcon from '@material-ui/core/ListItemIcon';
 import ListItemText from '@material-ui/core/ListItemText';
 import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+import SearchIcon from '@material-ui/icons/Search';
```

Comment:
It's kinda cool and all that our linter comments on PRs but...

Can we just
1. Run lint on commit (with husky and lint-staged)
2. Run lint on PR builds

Number 1 should realistically prevent 99% of these lint comments from showing up.